.ci/azure-pipelines/documentation.yml

```diff
@@ -24,15 +24,35 @@ jobs:
           addToPath: true
       - script: |
           mkdir $BUILD_DIR && cd $BUILD_DIR
+          cmake $(Build.SourcesDirectory) \
+            -DCMAKE_C_COMPILER=$(which clang) \
+            -DCMAKE_CXX_COMPILER=$(which clang++) \
+            -DCMAKE_EXPORT_COMPILE_COMMANDS=ON \
+            -DBUILD_CUDA=ON \
+            -DBUILD_GPU=ON \
+            -DBUILD_apps=ON \
+            -DBUILD_examples=ON \
+            -DBUILD_global_tests=ON \
+            -DBUILD_outofcore=ON \
+            -DBUILD_people=ON \
+            -DBUILD_simulation=ON \
+            -DBUILD_surface_on_nurbs=ON \
+            -DBUILD_visualization=ON
+        displayName: 'CMake Configuration for compilation database'
+      - script: |
+          cd $BUILD_DIR
+          cmake --build . --parallel ${nproc}
+        displayName: 'Build compilation database'
```

Comment:
What does this step try to achieve? To the best of my knowledge, the compilation database is an artifact of the configuration step, building is not required.

Reply:
I had never realized it's ready after configuration, thanks.